ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/hub-side-templating/configMaps/group-zones-configmap.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: ztp-group
 data:
   # ClusterLogForwarder.yaml
-  zone-1-cluster-log-fwd-outputs: "[{\"type\":\"kafka\", \"name\":\"kafka-open\", \"url\":\"tcp://10.46.55.190:9092/test\", \"secret\":{\"name\":\"kafka-open-secret\"}}]"
-  zone-1-cluster-log-fwd-pipelines: "[{\"inputRefs\":[\"audit\", \"infrastructure\"], \"labels\": {\"label1\": \"test1\", \"label2\": \"test2\", \"label3\": \"test3\", \"label4\": \"test4\"}, \"name\": \"all-to-default\", \"outputRefs\": [\"kafka-open\"]}]"
+  zone-1-cluster-log-fwd-outputs: "[{\"name\":\"default-lokistack\",\"type\":\"lokiStack\",\"lokiStack\":{\"target\":{\"name\":\"logging-loki\",\"namespace\":\"openshift-logging\"},\"authorization\":{\"token\":{\"from\":\"serviceAccount\"}}},\"tls\":{\"ca\":{\"key\":\"service-ca.crt\",\"configMapName\":\"openshift-service-ca.crt\"}}}]"
+  zone-1-cluster-log-fwd-pipelines: "[{\"name\":\"all-to-default\",\"inputRefs\":[\"applications\",\"audit\",\"infrastructure\"],\"outputRefs\":[\"default-lokistack\"]}]"
```

Comment:
Same as above. No loki. Only audit and infra logs.

Reply:
Got that in the latest changes